internal/backend/webhookssvc/svc.go

```diff
@@ -39,7 +39,7 @@ func (s *Service) Start(muxes *muxes.Muxes) {
 }
 
 func InitTrigger(trigger sdktypes.Trigger) sdktypes.Trigger {
-	return trigger.WithWebhookSlug(shortuuid.DefaultEncoder.Encode(sdktypes.NewUUID()))
+	return trigger.WithWebhookSlug(typeid.Must(typeid.WithPrefix("")).String())
```

Comment:
nope - use sdktypes.NewUUID here because it supports the idgen capability (preknown ids)

Reply:
done, ptal